<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Text</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="2">
      <widget class="QLineEdit" name="textLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Font-size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="colorLineEdit">
       <property name="text">
        <string>rgb(255,0,0)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="sizeLineEdit">
       <property name="text">
        <string>4pt</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="familyLineEdit">
       <property name="text">
        <string>Verdana</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Family</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="placeButton">
     <property name="text">
      <string>Place Text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>placeButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
